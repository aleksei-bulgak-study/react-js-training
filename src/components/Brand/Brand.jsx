import React from 'react';
import './Brand.css';

const Brand = () => (
  <p className="brand">
    <span className="brand__name">netflix</span>
    roulette
  </p>
);

export default Brand;
