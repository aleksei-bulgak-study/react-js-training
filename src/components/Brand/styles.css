.brand {
  height: 100%;
  display: flex;
  align-items: center;
  color: var(--red);
  font-weight: 200;
  font-size: 1.5rem;
}

.brand__name {
  font-weight: 800;
}
