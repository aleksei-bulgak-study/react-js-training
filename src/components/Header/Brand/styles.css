.brand {
  height: 100%;
  display: flex;
  align-items: center;
}

.brand-title {
  color: var(--red);
  position: relative;
  font-weight: 200;
  font-size: 1.5rem;
}

.brand-title__name {
  font-weight: 800;
}
