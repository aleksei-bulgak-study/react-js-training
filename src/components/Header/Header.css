/* header section */
header {
  position: absolute;
  top: 0;
  left: 6%;
  background-color: transparent;
  z-index: 3;
  height: 5vh;
}
