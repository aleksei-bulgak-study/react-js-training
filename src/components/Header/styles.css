/* header section */
header {
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  background-color: transparent;
  z-index: 2;
  height: 5vh;
}
