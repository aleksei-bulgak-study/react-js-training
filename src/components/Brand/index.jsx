import React from 'react';
import './styles.css';

const Brand = () => (
  <p className="brand">
    <span className="brand__name">netflix</span>
    roulette
  </p>
);

export default Brand;
