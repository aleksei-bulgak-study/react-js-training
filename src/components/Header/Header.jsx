import React from 'react';
import ErrorBoundary from '../ErrorBoundary';
import BrandHeader from '../Brand';
import './Header.css';

const Header = () => (
  <header>
    <ErrorBoundary>
      <BrandHeader />
    </ErrorBoundary>
  </header>
);

export default Header;
