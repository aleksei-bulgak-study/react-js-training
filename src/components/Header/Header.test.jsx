import React from 'react';
import { mount } from 'enzyme';

import Header from './Header';
import Brand from '../Brand';

describe('when Header', () => {
  test('then Header and Brand is rendered', () => {
    const header = mount(<Header />);
    expect(header).toMatchSnapshot();
    expect(header.containsMatchingElement(<Brand />)).toEqual(true);
  });
});
