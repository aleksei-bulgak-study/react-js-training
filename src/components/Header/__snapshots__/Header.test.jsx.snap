// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when Header then Header and Brand is rendered 1`] = `ReactWrapper {}`;
